docs/crypto.md

```diff
@@ -36,6 +36,20 @@ It takes one of the values for its first param:
 - `dsa`: Generate a DSA key (L2048N256)
 - `rsa`: Generate an RSA 4096 key
 
+## getCA
```

Comment:
idk about calling the function `getCA`. There's nothing CA-specific about what it does. It could be _any_ cert, right?